debian/control

```diff
@@ -54,11 +54,14 @@ Architecture: any
 Depends: adduser,
          apparmor (>= 2.10.95-0ubuntu2.2),
          ca-certificates,
+         cgroup-lite,
```

Comment:
This is a universe package in xenial, we either need a SRU or we need to add it only on trusty. Plus it pulls in mountall which is universe as well and not something we want in the default install AIUI (needs double checking with foundations).